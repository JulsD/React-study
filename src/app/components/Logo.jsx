import React from 'react';
import styles from './../../styles/logo.css';

class Logo extends React.Component {

  render() {
    return (
      <div className = {styles.root}>
        <img src='../img/logo.png' alt='netflixroulette' />
      </div>
    );
  }

}

export default Logo;
