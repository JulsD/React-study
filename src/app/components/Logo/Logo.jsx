import React from 'react';
import { Link } from 'react-router-dom'
import styles from './logo.css';

class Logo extends React.Component {

  render() {
    return (
      <div className = {styles.root}>
        <Link to='/'>
          netflixroulette
        </Link>
      </div>
    );
  }

}

export default Logo;
