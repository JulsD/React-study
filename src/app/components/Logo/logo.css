.root {
  color: #f40612;
  font-weight: bold;
  cursor: pointer;
}
.root a {
  color: inherit;
  font-weight: inherit;
  text-decoration: none;
}
