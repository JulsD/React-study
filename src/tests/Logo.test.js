import React from 'react';
import Logo from '../app/components/Logo';

test('Logo should render a <div>', () => {
  const wrapper = shallow(
    <Logo />
  );
  expect(wrapper).toMatchSnapshot();
});
