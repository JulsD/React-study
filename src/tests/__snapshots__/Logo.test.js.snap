// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render a <div> 1`] = `
<div
  className="root"
>
  <Link
    replace={false}
    to="/"
  >
    netflixroulette
  </Link>
</div>
`;
